### DISCUSSION TITLE
Noob question about committing on Windows 10

### DISCUSSION THREAD

**Discussion by Zamachi** [NeogitOrg/neogit#214] [Q&A] [answered]
Hi guys, I'm on Windows 10 *sigh*, I saw that the default command for actually doing a Commit, as in Magit, is practically the same C-c C-c? Or C-C C-C? No matter, whichever I tried, the commit was not made, `:messages` only displays a "hint" of sorts, saying: "use :qa! to abandon the changes", which is not what I need...

I haven't specifically changed any keys/mappings, only made the global leader key to be space(" "), which I'm using for which-key.
Other than that, I don't see why this wouldn't work...I have Git bash installed, it already is in the system PATH, funnily enough this works when I use it Emacs(yes, I realize this is a different plugin for a different editor).

**Comment by TimUntersberger** [comment, marked as answer]
@Zamachi I'm sorry! I forgot to update the documentation regardin this.

You can commit by typing `:wq` in the commit buffer.

**Comment by Zamachi** [comment]
Oh, so it wasn't me doing anything wrong ,alright!
Speaking of which, is there a way to *actually* bind the Commit to C-c C-c, because I've grown used to it in Emacs?

**Reply by TimUntersberger** [reply]
```vim
autocmd FileType NeogitCommitMessage nnoremap <buffer> <C-c><C-c> :wq<CR>
```

**Reply by Zamachi** [reply]
Thank you so much!
